tox.ini

```diff
@@ -51,7 +51,6 @@ envlist =
 # pymongo does not yet support Python 3.8: https://github.com/pymssql/pymssql/issues/586
 # but these tests still work.
     pymongo_contrib-py{38,39,310}-pymongo{30,31,32,33,35,36,37,38,39,310,}-mongoengine
-    pymysql_contrib-py{27,35,36,37,38,39,310}-pymysql{07,08,09,}
```

Comment:
It seems the new riot config would not test all these Python versions. So my simplification on those lines would not be possible and we need to correct the min/max versions.